landlab/components/weathering/exponential_weathering.py

```diff
@@ -109,17 +109,23 @@ def __init__(
         ----------
         grid: ModelGrid
             Landlab ModelGrid object
-        soil_production__maximum_rate : float
+        soil_production__maximum_rate : float, array of float
```

Comment:
What do you think about removing the double underscore from  the *soil_production__maximum_rate* and *soil_production__decay_depth* keywords? I want to be careful not to break existing code but I find the double underscores to be confusing.

Reply:
👍